app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
```

Comment:
Don't use `\` for separating a long line, you can move each argument to new line
```suggestion
    def __init__(
        self, 
        comfort_class: int, 
        clean_mark: int, 
        brand: str,
    ) -> None:
```